.github/workflows/scripts/linux/install-packages.sh

```diff
@@ -24,9 +24,9 @@ declare -a PCSX2_PACKAGES=(
   "libpng-dev"
   "libpulse-dev"
   "librsvg2-dev"
-  "libsdl2-dev"
```

Comment:
Wouldn't we still need sdl 2 in the list of libraries?

Reply:
We're compiling it ourselves now because the package manager's one is too old

Reply:
Ah, alright. Arch has a new enough version, so I hadn't noticed. Think we're probably good to merge...